dataeng/resources/monitor-bigquery-loading.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+env
+
+VENV_ROOT=$WORKSPACE/venvs
+mkdir -p $VENV_ROOT
+
+virtualenv $VENV_ROOT/analytics-tools
+TOOLS_BIN=$VENV_ROOT/analytics-tools/bin
+. $TOOLS_BIN/activate
+
+pip install -r ${WORKSPACE}/analytics-tools/bigquery-monitoring/requirements.txt
+
+# download credentials file to temporary directory
+aws s3 cp $CREDENTIALS /tmp/bigquery_credentials.json
```

Comment:
I don't like that creds are being copied into /tmp, then not cleaned up.  I'm going to rework this.

Reply:
The script that I used to load data into snowflake also required pulling credentials info off a S3 url.
I used boto to read the file contents from the s3 file, maybe we can do the same and pull the object in the python script. I'm also wondering if we need multiple script files.

Reply:
If we wanted to just make it one script file in the analytics-tools PR (just .py instead of .py + .sh) then we'd also need to leverage boto to do the `aws cloudwatch put-metric-data` step in addition to the `aws s3 cp`.  Consolidating into one python script is also my preference---any objection?